package.json

```diff
@@ -68,7 +68,7 @@
     "igniteui-angular": "^19.1.0-rc.0",
     "igniteui-angular-charts": "^19.0.0",
     "igniteui-angular-core": "^19.0.0",
-    "igniteui-angular-extras": "^19.0.3",
+    "igniteui-angular-extras": "file:../../igniteui-angular-extras-0.0.0.tgz",
```

Comment:
TODO: update linked test version when official extras package version is released